hamocc/mo_param_bgc.F90

```diff
@@ -728,8 +739,43 @@ subroutine write_parambgc()
       write(io_stdo_bgc,*) '*   opaldens     = ',opaldens
       write(io_stdo_bgc,*) '*   calcdens     = ',calcdens
       write(io_stdo_bgc,*) '*   claydens     = ',claydens
+      call add_entry('orgdens',orgdens,'','')
+      call add_entry('calcdens',calcdens,'','')
+      call add_entry('claydens',claydens,'','')
```

Comment:
A more descriptive name for the routine might be better? E.g. `pinfo_add_entry`

Reply:
Agree, I'll adopt your suggestion (as written initially, I also didn't like the naming).